Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -1589,6 +1589,8 @@ class RenderStyle {
     inline SVGPaintType fillPaintType() const;
     inline StyleColor fillPaintColor() const;
     inline void setFillPaintColor(const StyleColor&);
+    inline void setHasExplicitlySetFillColor(bool);
+    inline bool hasExplicitlySetFillColor() const;
```

Comment:
This should probably be `setHasExplicitlySetColor` / `hasExplicitlySetColor` to match the CSS property name.

Same applies to other instances of the name (except the one in `TextPaintStyle`).